docs/user_guide.md

```diff
@@ -1,9 +1,52 @@
 @mainpage
 
-## UFS_UTILS
+# UFS_UTILS
 
 Utilities for the NCEP models. This is part of the
 [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS) project.
 
-The UFS_UTILS code here: https://github.com/NOAA-EMC/UFS_UTILS.
+The UFS_UTILS code can be found here:
+https://github.com/NOAA-EMC/UFS_UTILS.
+
+## The Utilities
+
+- chgres_cube - Creates cold start initial conditions for FV3 model
+  runs.
+
+- emcsfc_ice_blend - Blends National Ice Center sea ice cover and EMC
+  sea ice concentration data to create a global sea ice analysis used
+  to update the GFS once per day.
+
+- emcsfc_snow2mdl - Blends National Ice Center snow cover and Air
+  Force snow depth data to create a global depth analysis used to
+  update the GFS snow field once per day.
+
+- fre-nctools - Tools to remap data; and to create the geo-reference
+  fields (latitude, longitude, etc.) for an FV3 grid.
+
+- fvcom_tools - - Replaces lake surface and lake ice temperature along
+  with aerial ice concentration generated from the Great Lakes
+  Operational Forecast System (GLOFS) in an FV3 surface restart
+  file. [fvcom documentation](sorc/fvcom_tools.fd/fvcom_readme.md)
```

Comment:
The links to the "fvcom_readme.md" and "global_chgres_user_guide.md" did not work for me.  They are also pointing to your branch.  After the merge, are they automatically updated to point to 'develop'?

Reply:
Hmmm. Let me check that...

Reply:
OK, fixed it. It doesn't work great in markdown, but the doxygen output is good. This is the same answer Jeff came up with on NCEPLIBS-bufr for dealing with markdown files in the doxygen docs.